### ISSUE TITLE
Do we want a production branch

### ISSUE THREAD

**Issue by ekrebs5** [hpi-swt2/workshop-portal#693]
If not, we should remove all references to it from the code

**Comment by bjrne** [comment]
I'd say yes, but in the sense that its HEAD is the currently rolled out version, so for now we wouldn't really need it

**Comment by cmfcmf** [comment]
> I'd say yes, but in the sense that its HEAD is the currently rolled out version, so for now we wouldn't really need it

Yes, that's also what I would suggest and what the people at [evap](https://github.com/fsr-itse/EvaP) are doing.
